packages/react-devtools-scheduling-profiler/README.md

```diff
@@ -1,3 +1,15 @@
 # Experimental React Concurrent Mode Profiler
 
-- Deployed at: https://react-scheduling-profiler.vercel.app
+https://react-devtools-scheduling-profiler.vercel.app/
+
+## Setting up continuous deployment with CircleCI and Vercel
+
+These instructions are intended for internal use, but may be useful if you are setting up a custom production deployment of the scheduling profiler.
+
+1. Create a Vercel token at https://vercel.com/account/tokens.
```

Comment:
These tokens make me a little uneasy, as they allow access to the whole Vercel account and we can't scope them to a single project. I don't think there's any alternative though.